bin/dataduct

```diff
@@ -126,57 +116,99 @@ class _HelpAction(argparse._HelpAction):
             for choice, subparser in subparsers_action.choices.items():
                 print "Command '{}'".format(choice)
                 print subparser.format_usage()
-
         parser.exit()
 
 
+def choice_help_formatter(choices):
+    """Create help format for a choice argument
+    """
+    result = []
+    for key, value in choices.iteritems():
+        result.append('%s: %s' % (key, value))
+    return '\n'.join(result)
+
+
 def main():
-    """Main function"""
-    parser = argparse.ArgumentParser(description='Run Dataduct commands',
-                                     add_help=False)
-    parser.add_argument(
+    """Main function
+    """
+
+    # Overwrite default help
+    help_parser = argparse.ArgumentParser(
+        description='Help message',
+        add_help=False)
+    help_parser.add_argument(
+        '-h',
+        '--help',
+        action=_HelpAction,
+        help='Help message',
+    )
+
+    # Base parser for all commands
+    base_parser = argparse.ArgumentParser(
+        description='Base Parser',
```

Comment:
Is this base parser doing anything? It doesn't seem to be connected to `help_parser` or `parser`. No parent, no children.